src/main/kotlin/com/terraformation/backend/customer/api/GlobalRolesController.kt

```diff
@@ -39,7 +40,19 @@ class GlobalRolesController(
       @PathVariable("userId") userId: UserId,
       @RequestBody payload: UpdateGlobalRolesRequestPayload,
   ): SuccessResponsePayload {
-    userStore.updateGlobalRoles(userId, payload.globalRoles)
+    userStore.updateGlobalRoles(setOf(userId), payload.globalRoles)
+
+    return SimpleSuccessResponsePayload()
+  }
+
+  @ApiResponse200
+  @ApiResponse404
+  @DeleteMapping("/users/globalRoles")
```

Comment:
I think this might be better as `/globalRoles/users` so that (a) it's not defining another special literal user ID under `/api/v1/users` in addition to "me" and (b) because it is kind of symmetrical to `GET /api/v1/globalRoles/users` in that it operates on lists of users that have global roles.

Reply:
I wrestled with that for a moment, ended up going with what I did because I generally try to put "the resource" (user.globalRoles) in question at the end. I will change it up.

Reply:
I generally try to put the resource (I know we aren't RESTful, but I can't help myself) at the end. But you make a good point about the new special literal ID, I'll move it.